docs/user_guide/3_getting_started.md

```diff
@@ -27,34 +40,50 @@ We have created a one-liner that is able to install Docker, download the image a
   ```
   In the above example, ‘enp0s25’ is the interface ID that is needed.
 
+* **Get AWS login credentials**
+
+  If you want to upload technical logged data (ROS logs, backtraces, crash dumps etc.) to AWS server and notify the Shadow's software team to investigate your bug then you need to enable AWS uploading in the one-liner. In order to use this option you need to obtain a unique AWS key by emailing sysadmin@shadowrobot.com. When you receive the key you can use it when running the one-liner installation tool. To enable the AWS uploading you need to add the command line option ```-ck true``` to the one-liner.
+
 * **Check your hand configuration branch**:
 
   You should have the name of your [sr_config](https://github.com/shadow-robot/sr-config) hand branch which contains the specific configuration of your hand (calibration, controller tuning etc…).
   Usually it is something like this: ``shadowrobot_XXXXX``. 
 
-  You can check which branch is installed on the computer provided by Shadow by running (on the machine provided with your hand):
-  ```bash
-  $ roscd sr_ethercat_hand_config
-  $ git branch
-  ```
-  The highlighted branch is the one that is currently used. 
   If you are unsure please contact us. 
 
 * **Run the one-liner**:
 
-  The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, use the following command:
+  The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, use the following command (please remember to replace [EtherCAT interface ID] with your Interface ID and [sr_config_branch] with your unique sr_config branch):
```

Comment:
Please note section should be more visible

Reply:
done